lib/build-bundler.js

```diff
@@ -52,13 +52,14 @@ function buildSassRenderer(options) {
 		includePaths,
 		sourceMapEmbed: sourcemaps,
 		functions: {
-			"asset-url($assetName)": (assetName, done) => {
-				assetManager.manifest.resolve(assetName.getValue()).
-					then(mappedAssetName => {
-						done(new SassString(`url("${mappedAssetName}")`));
-					}).catch(err => {
-						done(new SassError(err.message));
-					});
+			"asset-url($assetName)": assetName => {
```

Comment:
The done parameter is optional. If it is provided, the function is treated as async, otherwise it is treated as sync. We no longer need to be async here, so I removed it.